requirements.txt

```diff
@@ -1,4 +1,5 @@
-dbt-clickhouse==1.7.3
+dbt-clickhouse==1.7.4
 dbt-core==1.7.10
 shandy-sqlfmt[jinjafmt]==0.21.2
 dbt-coverage
+clickhouse-cityhash
```

Comment:
This allows us to use compression when connecting to ClickHouse, which also allow us to use the ClickHouse client port which we should use by default.